aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,373 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
+
+Generating good documentation and help pages as a part of your Web API using `Swagger <a href="http://swagger.io/">`_ with the .NET Core implementation `Swashbuckle <a href="https://github.com/domaindrivendev/Ahoy">`_ is as easy as adding a couple of NuGet packages and modifying the *Startup.cs*.
```

Comment:
We need to clarify that Swashbuckle is a 3rd party community package. Something like "Swashbuckle is a community package for generating Swagger documents for Web APIs built with ASP.NET Core MVC"